kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/parquet/ParquetFileWriterSuite.scala

```diff
@@ -58,7 +58,7 @@ class ParquetFileWriterSuite extends AnyFunSuite
 
   Seq(
     // Test cases reading and writing all types of data with or without stats collection
-    Seq((200, 100), (1024, 28), (1048576, 1)).map {
+    Seq((200, 100), (1024, 29), (1048576, 1)).map {
```

Comment:
this change is due to number of files changed due to the new column that is added to the test.